features/step_definitions/networking.rb

```diff
@@ -1590,3 +1590,13 @@
   end
   logger.info "The egressfirewall type is applied."
 end
+
+Given /^the cluster is dual stack network type$/ do
+  ensure_admin_tagged
+  @result = admin.cli_exec(:get, resource: "network.operator", output: "jsonpath={.items[*].spec.serviceNetwork}")
+  unless @result[:response].count(":") >= 2 && @result[:response].count(".") >= 2  
+    logger.warn "the cluster is not dual stack cluster, will skip this scenario"
+    logger.warn "We will skip this scenario"
```

Comment:
@zhaozhanqi Duplicate with above line ?

Reply:
oh, bad copy :).  thanks for catching this. 
fixed